test/e2e/generator/generate.go

```diff
@@ -27,8 +27,9 @@ var (
 	}
 
 	// The following specify randomly chosen values for testnet nodes.
+	// TODO settle for one DB only
 	nodeDatabases         = uniformChoice{"goleveldb", "cleveldb", "rocksdb", "boltdb", "badgerdb"}
```

Comment:
What is the preferred / fastest DB again? 
cc @marbar3778 @tzdybal

Reply:
I would prefer to go with a DB with acid support, Badger is a good candidate for this. We should separate the db work into another PR, that way we can add custom configurations to help optimize for our needs. Ill open an issue.

Reply:
I've made the e2e tests already only use badger in https://github.com/lazyledger/lazyledger-core/pull/208/commits/4e03ca13d9b36979308b4ac8601c3abf727d2e50

It makes sense to rip out the broader DB story in a separate PR, though. Thanks for opening #211 🙏🏼